src/test/java/seedu/address/model/module/ModuleTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.module;
```

Comment:
Isit possible to rename the file to ModuleCodeTest.java? Just so the naming is consistent with the Class Name, thanks

Reply:
Alright